operator/pkg/tpath/struct_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tpath
+
+import (
+	"github.com/ghodss/yaml"
+
+	"testing"
+
+	"istio.io/istio/operator/pkg/util"
```

Comment:
As per Go convention, this is the order of imports

i) stdlib
ii) third-party imports
iii) first-party imports
```suggestion
	"testing"

	"github.com/ghodss/yaml"

	"istio.io/istio/operator/pkg/util"
```